VisualStudio.gitignore

```diff
@@ -297,6 +297,19 @@ paket-files/
 __pycache__/
 *.pyc
 
+# Python virtual environments
```

Comment:
Why not use `Python.gitignore` which has these rules?

Reply:
When a new project is created in Visual Studio with an option selected for creating a git repository, VS specifically uses VisualStudio.gitignore to initialize the .gitignore for that repo, regardless of the language used in the project. (More specifically, each release of VS copies VisualStudio.gitignore during its build process, and that file is what's used for .gitignore during new project creation.) Because VS does not provide a user with the option to select a .gitignore base or to customize it, and because we want to exclude venvs from new Python projects by default, we need to have this change in VisualStudio.gitignore. Does that make sense? 

As a further background, we're trying to address a behavioral issue with VS and Python when creating a git repository when creating a project. By default, VS is set up to do an auto-commit of the project after it's created. However, creating a project from a Python template in VS also typically creates a venv at the same time, which ends up being committed unnecessarily to source control. The steps to remove it from source control requires one to use CLI git commands, rather than being able to remain in VS. 

There are cumbersome workarounds for this, but the simplest answer is this small modification to VisualStudio.gitignore so that we get the right behavior by default.

Reply:
Interesting.  And IMO, that situation makes it worth the duplication.  Perhaps document the reason for it as a comment in the source file.